src/components/ShopOnMap.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <div class="columns">
+        <h3 v-if='location' >User location coords:  {{location[0]}} {{location[1]}}</h3>
+        <gmap-map
+            :center='shopCoords'
+            :zoom='zoom'
+            class='map_shop'
+        >
+            <gmap-marker
+                :key=''
+                :position='shopCoords'
+                :clickable=true
+                :draggable=false
+                @click='shopClicked'
+            >
+            </gmap-marker>
+        </gmap-map>
+        <div></div>
+    </div>
+</template>
+
+<script>
+import * as VueGoogleMaps from 'vue2-google-maps';
+import Vue from 'vue';
+
+Vue.use(VueGoogleMaps, {
+    load: {
+        key: 'AIzaSyB-c3v8sMxEM5nDACJKUFBH4p_Up8eaFUs',
```

Comment:
this isn't production level and we are not getting the key from the server as We will, so this is on this level and of course this should be added as another task connected as back + front